modules/docs/arrow-docs/docs/_data/commons.yml

```diff
@@ -6,3 +6,4 @@ google_analytics: "UA-18433785-18"
 github_repository: "https://github.com/arrow-kt/arrow"
 stable_version: "https://arrow-kt.io/"
 next_version: "https://arrow-kt.io/docs/next/"
+previous_version: "https://arrow-kt.io/docs/0.9/"
```

Comment:
This line is not needed, `previous_version` is declared in the `_head-docs.html` file, and it takes it from the previous title of the `stable_version` from `doc-versions.yml` file, so we can remove this line.
Doing this way when we do the changes to a new version we only need to change the `doc-versions.yml` file

![Captura de pantalla 2020-02-20 a las 10 38 17](https://user-images.githubusercontent.com/25897490/74920867-21482a00-53cd-11ea-8d3b-dc58ec7c3cd7.png)